### PR TITLE
Fix gathering apiserver profiles

### PR DESCRIPTION
**Pull request by mm4tt**
Fixes https://github.com/kubernetes/perf-tests/issues/985

### CODE REVIEW

**Comment by mm4tt** [comment]
/hold 

Let's verify the presubmits

**Comment by mm4tt** [comment]
/hold cancel
Profiles look good

File: clusterloader2/pkg/measurement/common/profile.go [hunk 1]
```diff
@@ -91,6 +95,12 @@ func (p *profileMeasurement) start(config *measurement.MeasurementConfig) error
 		klog.Warning("Profile measurements will be disabled due to no MasterIps")
 		return nil
 	}
+	c := config.ClusterFramework.GetClientSets().GetClient()
```
Comment (jkaniuk) [inline clusterloader2/pkg/measurement/common/profile.go]: Any reason to put it so early in execution?
It is hard to look for "c" so maybe I am missing some use of this client. Could you use more descriptive name?

Comment (mm4tt) [reply clusterloader2/pkg/measurement/common/profile.go]: Reason is that it's used a few lines below.

I renamed to k8sClient to make it more descriptive, thanks!

Comment (jkaniuk) [reply clusterloader2/pkg/measurement/common/profile.go]: Thx. Now I see it :)

**Pull request merged**